supportedBackends/htcondor/src/main/scala/cromwell/htcondorbackend/HtCondorBackend.scala

```diff
@@ -0,0 +1,29 @@
+package cromwell.htcondorbackend
```

Comment:
I was worried that naming them `cromwell.backend.<backendname>` would be confusing as we already have `cromwell.backend` for the global backend project containing backend definition